sunkit_pyvista/plotter.py

```diff
@@ -127,9 +127,9 @@ def plot_map(self, m, **kwargs):
         **kwargs :
             Keyword arguments are handed to `pyvista.Plotter.add_mesh`.
         """
-        cmap = kwargs.pop('cmap', m.cmap)
+        kwargs.pop('cmap', m.cmap)
```

Comment:
What was the reason for this change?

Reply:
My bad, that wasn't supposed to be a thing. I did that to demonstrate how a plot without cmap would look.